### DISCUSSION TITLE
New font ?

### DISCUSSION THREAD

**Discussion by clementwzk** [clementwzk/OpenCalc#354] [Ideas]
Do you think we should change the font to something more elegant?  If so, do you have any ideas?  We could also add an option to be able to change the font

**Comment by inson1** [comment]
We could add option to select font from x fonts, that we know are alright

But I like the font we have now.

**Comment by clementwzk** [comment]
the most important thing is to choose the default font

**Reply by inson1** [reply]
@Darkempire78 You dont like our current font?

**Reply by clementwzk** [reply]
not especially

**Comment by clementwzk** [comment]
Do you have new font to suggest ?

**Reply by F7640BEVUD** [reply]
I would suggest the Google Sans Display font, which is available for download [here](https://github.com/mobiledesres/Google-UI-fonts/blob/main/zip/Google%20Sans%20Display.zip). Google uses this font on its Google Calculator application for the buttons. I am not sure what font style they use for the math expressions, however.

You can implement this font as the default font for the application or perhaps develop a way for users to manually import an external fonts file. You can take inspiration from the Read You app found [here](https://github.com/Ashinch/ReadYou), which has code for manual font file imports.
